lib/feedPage.dart

```diff
@@ -0,0 +1,713 @@
+  bool isSwitched4 = true;
+  bool isSwitched5 = true;
+  bool isSwitched6 = true;
+  bool isSwitched7 = true;
+  bool isSwitched8 = true;
+  bool isSwitched9 = true;
+  bool isSwitched10 = true;
+  bool isSwitched11 = true;
+  bool isSwitched12 = true;
+  bool isSwitched13 = true;
+  bool isSwitched14 = true;
+  bool isSwitched15 = true;
+  bool isSwitched16 = true;
+  bool isSwitched17 = true;
+  bool isSwitched18 = true;
+  bool isSwitched19 = true;
+  bool isSwitched20 = true;
+  bool isSwitched21 = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: MediaQuery.of(context).size.width * 0.7,
+      child: Drawer(
+        child: Column(
+          children: [
+            DrawerHeader(
+              child: GestureDetector(
+                onTap: () {
+                  Navigator.pushNamed(context, '/third');
+                },
+                child: Container(
+                  decoration: BoxDecoration(
+                    shape: BoxShape.circle,
+                    boxShadow: [
+                      BoxShadow(
+                        blurRadius: 8.0,
+                        color: Colors.black54,
+                        spreadRadius: 0.9,
+                      )
+                    ],
+                  ),
+                  child: CircleAvatar(
+                    radius: 70.0,
+                    backgroundImage: AssetImage("assets/profilePic.jpg"),
+                    backgroundColor: Colors.black,
+                  ),
+                ),
+              ),
+              // decoration: BoxDecoration(
+              //     image: DecorationImage(
+              //       image: AssetImage("assets/app_logo_final0.png"),
+              //       fit: BoxFit.fitHeight,
+              //     )),
+            ),
+            Center(
+              child: Container(
+                color: Color(0xFF181D3D),
+                child: ListTile(
+                  title: Center(
+                    child: Text(
+                      'Hi, Kenny ', // declare the variable for the name of user
+                      style: TextStyle(
+                        fontSize: 25.0,
+                        fontFamily: 'JosefinSans',
+                        color: Colors.white,
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+            ),
+            Expanded(
+              child: ListView(
+                padding: EdgeInsets.all(2.0),
+                children: [
+                  ExpansionTile(
+                    leading: Icon(
+                      Icons.filter_list,
+                      color: Color(0xFF181D3D),
+                    ),
+                    title: Text(
+                      'Category',
+                      style: TextStyle(
+                        fontSize: 15.0,
+                      ),
+                    ),
+                    children: [
+                      ListTile(
```

Comment:
Since you have all similar ListTile widgets here, you could have used a function to generate these avoiding hardcore programming.

Reply:
Okay, while merging the codes, I will refine this page.